### ISSUE TITLE
"App with id: ###### not found" message is scary

### ISSUE THREAD

**Issue by pananapread** [aws-amplify/amplify-console#2195]
**Is your feature request related to a problem? Please describe.**

I'm always frustrated when I switch environments to check on another Amplify app and the banner is all red, scary, and obtrusive!

![image](https://user-images.githubusercontent.com/12780104/130453344-35ce9db0-7967-4b20-9a51-14176aa2892c.png)

Additionally, the difference between "connect app" and "new app" is unclear. Why can I at times create a new app and other times only connect? (I know it's because certain envs aren't supported yet, but maybe make this more clear)

![image](https://user-images.githubusercontent.com/12780104/130453405-9697c3e6-8828-4a2f-854f-de177871de60.png)

**Describe the solution you'd like**

We should communicate more clearly that
- When switching envs: nothing bad happened to your app, it's simply not in that particular environment - maybe make the banner more of an informational / more easily dismissible?
- Clarify on connects vs. new 

![image](https://user-images.githubusercontent.com/12780104/130453455-0a1c4262-b46d-4118-b852-dfc948901b2a.png)
![image](https://user-images.githubusercontent.com/12780104/130453497-72d38279-ea27-4d12-af04-da6e9f490f11.png)

**Additional context**
All images by @njreid

**Comment by swaminator** [comment]
@pananapread Agreed on the regions `appid not found issue`. However, the **new app** dropdown today says: "create app backend" and "Host web app". What specifically are you suggesting?

**Comment by pananapread** [comment]
I would like to see the same dropdown options even in regions where "create app backend" is not available (with an informational explaining why)

Also consistency in language -- rather than "Connect" it should be "Host web app"

![image](https://user-images.githubusercontent.com/12780104/131380969-6d20ea36-920a-4ece-a84f-d1c605a696e7.png)

![image](https://user-images.githubusercontent.com/12780104/131381108-fa400020-9d2d-461e-9a48-d1dd9d42f29f.png)
